reference-architecture/azure-ansible/allinone.json

```diff
@@ -0,0 +1,670 @@
+        "Standard_A11",
+        "Standard_D2",
+        "Standard_D3",
+        "Standard_D4",
+        "Standard_D11",
+        "Standard_D12",
+        "Standard_D13",
+        "Standard_D14",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_G1",
+        "Standard_G2",
+        "Standard_G3",
+        "Standard_G4",
+        "Standard_G5",
+        "Standard_DS2",
+        "Standard_DS3",
+        "Standard_DS4",
+        "Standard_DS11",
+        "Standard_DS12",
+        "Standard_DS13",
+        "Standard_DS14",
+        "Standard_DS2_v2",
+        "Standard_DS3_v2",
+        "Standard_DS4_v2",
+        "Standard_DS5_v2",
+        "Standard_DS11_v2",
+        "Standard_DS12_v2",
+        "Standard_DS13_v2",
+        "Standard_DS14_v2",
+        "Standard_GS1",
+        "Standard_GS2",
+        "Standard_GS3",
+        "Standard_GS4",
+        "Standard_GS5"
+      ],
+      "metadata" : {
+        "description" : "The size of the each Node Virtual Machine."
+      }
+    },
+    "rhsmUsernamePasswordOrActivationKey" : {
+      "type" : "string",
+      "minLength" : 1,
+      "defaultValue" : "usernamepassword",
+      "allowedValues" : [
+        "usernamepassword",
+        "activationkey"
+      ],
+      "metadata" : {
+        "description" : "Select whether you want to use your Red Hat Subscription Manager Username and Password or Organization ID and Activation Key to register the RHEL instance to your Red Hat Subscription."
+      }
+    },
+    "RHNUserName" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Red Hat Subscription User Name or Email Address or Organization ID"
+      }
+    },
+    "RHNPassword" : {
+      "type" : "securestring",
+      "metadata" : {
+        "description" : "Red Hat Subscription Password or Activation Key"
+      }
+    },
+    "SubscriptionPoolId" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Pool ID of the Red Hat subscritpion to use"
+      }
+    },
+    "sshPrivateData" : {
+      "type" : "securestring",
+      "metadata" : {
+        "description" : "SSH RSA private key file as a base64 string."
+      }
+    },
+    "aadClientId" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "Azure AD Client Id"
+      }
+    },
+    "aadClientSecret" : {
+      "type" : "securestring",
+      "metadata" : {
+        "description" : "Azure AD Client Secret"
+      }
+    },
+    "metrics" : {
+      "type" : "bool",
+      "defaultValue" : true,
+      "metadata" : {
+        "description" : "Enable OCP metrics"
+      }
+    },
+    "logging" : {
+      "type" : "bool",
+      "defaultValue" : true,
+      "metadata" : {
+        "description" : "Enable OCP aggregated logging"
```

Comment:
Making it default is overkill (3 elasticsearch pods + kibana + curator + fluentd) for an all-in-one.